encord/utilities/label_utilities.py

```diff
@@ -1,3 +1,15 @@
+"""
+---
+title: "Utilities - Labels"
+slug: "sdk-ref-utilities-labels"
+hidden: false
+metadata:
+  title: "Client Utilities"
+  description: "Encord SDK Client Utilities."
+category: "64e481b57b6027003f20aaa0"
+---
+"""
+
```

Comment:
This file is very legacy. I probably would leave it out unless we have some code examples or smth that absolutely require it

Reply:
I'll remove the header for now, and add it back if there are issues.